corehq/apps/userreports/util.py

```diff
@@ -33,18 +32,6 @@
 LEGACY_UCR_TABLE_PREFIX = 'config_report_'
 
 
-@dataclass
```

Comment:
Thinking about the roll out, how will this work for the updates queued but yet to fire?

Reply:
474a5ba9c83